timeline-chart/src/time-graph-model.ts

```diff
@@ -17,6 +17,7 @@ export namespace TimelineChart {
         name: string
         range: TimeGraphRange
         states: TimeGraphRowElementModel[]
+        annotations: TimeGraphAnnotation[]
```

Comment:
Should this be { [category: string]: TimeGraphAnnotation[] }?

Do we want to group annotations per category, or do we want to add the category in each annotation, or not have the category at all?

For context, the intended purpose of annotation category is 1) allow bulk show/hide per category 2) to be shown in tooltip 3) not supported yet but to only request and receive shown categories in TSP

Reply:
Should it not be done on the model side? I want to keep the view dumb.

Reply:
Yes, after discussion, timeline-chart doesn't need to know the category, any filtering on category or tooltip should be done by the user of the chart.